platform-apps/charts/backstage/templates/ingress.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.gardenercert.enabled }}
```

Comment:
From my understanding for other target types which have gardenercert.enabled false or not set no ingress gets created. Is that on purpose?